docs/building_the_project.rst

```diff
@@ -490,6 +514,12 @@ CMake.
   ``-DHIPSYCL_TARGETS`` according to the targeted hardware. For the options,
   see the tables in the hipSYCL-specific sections.
 
+
+.. note::
+  When building with clang++ for AMD backends, you must additionally provide
+  ``-DHIP_TARGETS`` according to the targeted hardware. For the options,
```

Comment:
Does this HIP_TARGETS used in LLVM compiler? I could not find it after a quick search.